README.md

```diff
@@ -74,9 +75,9 @@ options.includesSteps = true
 
 Directions.shared.calculate(options) { (waypoints, routes, error) in
     guard let route = routes?.first else { return }
-
-    let viewController = NavigationViewController(for: route)
-    viewController.simulatesLocationUpdates = true
+    
+    let locationManager = SimulatedLocationManager(route: route)
```

Comment:
@frederoni what's your thinking behind adding `SimulatedLocationManager` here? I'm sort of thinking we want to make the example that gets the most eyes, the most barebones example we can come up with.